### ISSUE TITLE
Dataset Download

### ISSUE THREAD

**Issue by yujialele** [hxixixh/mix-and-localize#7]
Thank you for your best work! I fail to download the datasets. Please provide the dataset download links.

**Comment by IFICL** [comment]
Hi, we have another link in the paper stay alive: https://web.eecs.umich.edu/~ahowens/mix-localize/

**Comment by laulliam** [comment]
@IFICL By the way, is there only a test dataset available?

**Comment by IFICL** [comment]
@laulliam Yes. We only develop an evaluation set. If you are looking for a training set with mask annotation, perhaps [AVSBench](http://www.avlbench.opennlplab.cn/dataset/avsbench) is something related.
